cmd/config/root.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/tani-yu/dogleash/cmd"
+)
+
+// dogleash config current and orgs
+type DogleashConfig struct {
+	Organizations []Organization
+	Current       string `mapstructure:"current"`
+}
+
+// orgs
+type Organization struct {
+	Name   string `mapstructure:"name"`
+	APIKey string `mapstructure:"apikey"`
+	APPKey string `mapstructure:"appkey"`
+}
+
+// DC dogleash config
+var DC DogleashConfig
```

Comment:
```suggestion
var DC cmd.DogleashConfig
```

Then, you can delete duplicated declarations of both DogleashConfig and Organization above.